integrations.go

```diff
@@ -0,0 +1,105 @@
+package sentry
```

Comment:
Modules will never change throughout the lifecycle of the app and we want to minimize number of I/O operations. Can we cache the result once it was called?

Reply:
How can it not change? Every time any dependency is added to the project. Modules will change of course and this can happen at any frequency.

Reply:
And every time you restart your app, it'll cache a new state :)

Reply:
Oh, in this way. Yes, makes sense! I will do the requested changes by tomorrow.